new-docs/puppeteer.keyboard.md

```diff
@@ -4,33 +4,58 @@
 
 ## Keyboard class
 
+Keyboard provides an api for managing a virtual keyboard. The high level api is , which takes raw characters and generates proper keydown, keypress/input, and keyup events on your page.

```

Comment:
hmmm, seems like API Extractor is missing a link here from the docs ("high level api is , which ..."). Not sure quite why that is.